examples/kubernetes/README.md

```diff
@@ -30,6 +31,30 @@ CLUSTER_CERT=/etc/srv/kubernetes/pki/apiserver.crt
 CLUSTER_KEY=/etc/srv/kubernetes/pki/apiserver.key
 ```
 
+# Register SERVER_TOKEN in [static-token-file](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#static-token-file)
```

Comment:
Is "static-token" the standard way to authenticate a process running in the master node?

I think we can run the proxy server as a static pod, and then use a service account to authenticate it.

Reply:
Yes, this pattern used across all other static pods. Ex: https://github.com/kubernetes/kubernetes/blob/c14106ad1234742da80eb8f12ddcbf19dba61284/cluster/gce/gci/configure-helper.sh#L613-L615